app/views/profiles/dashboard/_journey.html.erb

```diff
@@ -19,6 +19,13 @@
               <li> <i class="fa fa-money" aria-hidden="true"></i>  <%= t('.price') %>:€ <%= journey.car.price_hour.to_i %>/hr</li>
               <!-- <li> <i class="fa fa-clock-o" aria-hidden="true"></i>  <%#= t('.duration') %>:<%= journey.duration %> h </li> -->
               <li> <i class="fa fa-users" aria-hidden="true"></i  >  <%= t('.children') %>: <%= journey.remaining_seats %> </li>
+              
+              <% if journey.logs[0] %>
```

Comment:
Do exactly what I indicated - make `calculate_hours` check `journey.logs[0]`, returning different values depending on that condition

`h = calculate_hours`
`if h ...`